pandas/core/groupby/groupby.py

```diff
@@ -1191,7 +1191,9 @@ def std(self, ddof=1, *args, **kwargs):
 
         # TODO: implement at Cython level?
         nv.validate_groupby_func('std', args, kwargs)
-        return np.sqrt(self.var(ddof=ddof, **kwargs))
+        with _group_selection_context(self):
```

Comment:
why are you changing to use the python routine here, this makes this non-performant

Reply:
New to this code base, but I thought this was the way to fix; it seemed that this was the way to make sure that the operation was not applied to the grouped by column(s). I was patterning off of `var`.

Reply:
try this: https://github.com/pandas-dev/pandas/issues/10355#issuecomment-112015087

Reply:
@jreback Perfect, updated to use this but still falling back on a Python version if it doesn't work; if it's safe to just use the Cython version always let me know and I can remove the try/except block.